eos-update-flatpak-repos.service

```diff
@@ -28,5 +28,10 @@ Type=oneshot
 RemainAfterExit=true
 ExecStart=/usr/sbin/eos-update-flatpak-repos
 
+# Flatpak checks parental controls at deploy time. In order to do this, it
+# needs to talk to accountsservice on the system bus, neither of which are
+# running when this job runs.
+Environment=FLATPAK_SKIP_PARENTAL_CONTROLS_NO_SYSTEM_BUS=1
```

Comment:
Note: This environment variable was supposed to be a temporary workaround to unblock flatpak development. And yet, predictably, it’s now showing up all over the place to fix things. I don’t have a better expedient suggestion though.

Reply:
Yeah, I know…

I think we should revisit this and see if there is a way to implement the policy "uid 0 ⇒ no parental controls checks". But as you say, I don't think there is a better expedient solution.